src/overlays/actors/ovl_Dm_Char07/z_dm_char07.h

```diff
@@ -5,9 +5,31 @@
 
 struct DmChar07;
 
+typedef void (*DmChar07ActionFunc)(struct DmChar07*, GlobalContext*);
+
+#define DMCHAR07_GET_SPOTLIGHTFLAGS(thisx) (((thisx)->params >> 8) & 0x1F)
+#define DMCHAR07_GET_TYPE(thisx) ((thisx)->params & 0xFF)
+
+#define DMCHAR07_STAGE_RIGHT_SPOTLIGHT 1   // Deku spotlight
+#define DMCHAR07_UPSTAGE_RIGHT_SPOTLIGHT 2 // Goron spotlight
+#define DMCHAR07_STAGE_CENTER_SPOTLIGHT 4  // Human spotlight
+#define DMCHAR07_UPSTAGE_LEFT_SPOTLIGHT 8  // Zora spotlight
+#define DMCHAR07_DOWNSTAGE_CENTER_SPOTLIGHT 16
+
+typedef enum {
+    /* 0x00 */ STAGE,
+    /* 0x01 */ CREDITS_STAGE,
+    /* 0x02 */ SPOTLIGHTS,
+    /* 0x03 */ MISC_OBJECTS
```

Comment:
When defining enums for an actor, you should prefix the enum with the actor's name or an abbreviated version of the actors name. I think this is introducing a variable called `STAGE` across many files, anywhere with this header in it directly or indirectly, and there's a lot of places `STAGE` could mean